### PR TITLE
Remove mention of docker --bip flag.

### PR DESCRIPTION
**Pull request by erictune**

### CODE REVIEW

**Comment by erictune** [comment]
fixex #11167

File: docs/getting-started-guides/scratch.md [hunk 1]
```diff
@@ -336,7 +336,7 @@ brctl delbr docker0
 
 The way you configure docker will depend in whether you have chosen the routable-vip or overlay-network approaches for your network.
 Some docker options will want to think about:
-  - create your own bridge for the per-node CIDR ranges, and set `--bridge=cbr0` and `--bip=false`.  Or let docker do it with `--bip=true`.
+  - create your own bridge for the per-node CIDR ranges, and set `--bridge=cbr0`.  Or let docker do it with something like: `--bip=$NODE_X_POD_CIDR`.
```
Comment (thockin) [inline docs/getting-started-guides/scratch.md]: I think mentioning --bip is more complex that simply documenting our cbr0 flag to kubelet and making those instructions good.  For example, --bip assumes that docker0 does not exist, but if it does exist with a different IP, docker fails to start.

**Comment by erictune** [comment]
@thockin ptal

File: docs/getting-started-guides/scratch.md [hunk 2]
```diff
@@ -335,16 +335,15 @@ brctl delbr docker0
...
+  - create your own bridge for the per-node CIDR ranges, call it cbr0, and set `--bridge=cbr0` option on docker.
+  - set `--iptables=false` so docker will not manipulate iptables for host-ports (too coarse on older docker versions, may be fixed in newer versions)
 so that kube-proxy can manage iptables instead of docker.
   - `--ip-masq=false`
     - if you have setup PodIPs to be routable, then you want this false, otherwise, docker will
       rewrite the PodIP source-address to a NodeIP.
     - some environments (e.g. GCE) still need you to masquerade out-bound traffic when it leaves the cloud environment. This is very environment specific.
     - if you are using an overlay network, consult those instructions.  
-  - `--bip=`
     - should be the CIDR range for pods for that specific node.
```
Comment (thockin) [inline docs/getting-started-guides/scratch.md]: forgot to remove this line?

File: docs/getting-started-guides/scratch.md [hunk 3]
```diff
@@ -335,16 +335,15 @@ brctl delbr docker0
...
   - `--mtu=`
     - may be required when using Flannel, because of the extra packet size due to udp encapsulation
```
Comment (thockin) [inline docs/getting-started-guides/scratch.md]: Aside: I don't think anyone has tested flannel with our kubelet's cbr0 logic.

Comment (erictune) [inline docs/getting-started-guides/scratch.md]: noted.

Comment (dchen1107) [inline docs/getting-started-guides/scratch.md]: cc/ @bakins @yifan-gu, our coreos friends. :-)

File: docs/getting-started-guides/scratch.md (hunk 2 above)
Comment (erictune) [inline docs/getting-started-guides/scratch.md]: fixed.

Comment (thockin) [inline docs/getting-started-guides/scratch.md]: still true

**Comment by erictune** [comment]
PTAL

**Comment by thockin** [comment]
LGTM

**Comment by davidopp** [comment]
@k8s-bot test this please

**Pull request merged**